packages/dashboard-frontend/src/store/Workspaces/devWorkspaces/index.ts

```diff
@@ -505,7 +505,7 @@ export const actionCreators: ActionCreators = {
       const openVSXUrl = selectOpenVSXUrl(state);
       const defaultNamespace = defaultKubernetesNamespace.name;
       try {
-        const cheEditor = editorId ? editorId : devWorkspaceTemplateResource.metadata.name;
+        const cheEditor = editorId ? editorId : state.dwPlugins.defaultEditorName;
```

Comment:
Hello @olexii4 , I was wondering if it is okay to make this change? This helps determine the editor ID (ex. `che-incubator/che-code/latest`) of a devworkspace which is needed to determine default plugins from the CheCluster CR

I noticed that it was updated as part of https://github.com/eclipse-che/che-dashboard/pull/751  two months ago: https://github.com/eclipse-che/che-dashboard/commit/b229ff4598d2af3990a8daa0cd71bdf061ec740d#diff-c9344a8d3b76b9a96206044091ed1fff4924dbd53684a56ad0f8cb02a80f04fc